src/transactions/test/PaymentTests.cpp

```diff
@@ -274,7 +274,7 @@ TEST_CASE_VERSIONS("payment", "[tx][payment]")
             REQUIRE(expectedrootBalance == root.getBalance());
         });
 
-        for_versions_from(9, *app, [&] {
+        for_versions(9, 19, *app, [&] {
```

Comment:
This test can still be done if you change the fee source for `tx2` and set the source account on the operation to `b1` right?

Reply:
You're right, updated the test